app/src/main/java/de/tum/in/tumcampusapp/models/tumo/ObfuscatedIds.kt

```diff
@@ -4,6 +4,6 @@ import org.simpleframework.xml.Element
 import org.simpleframework.xml.Root
 
 @Root(name = "row")
```

Comment:
Why not just use `@Root(strict=false)` instead of adding it to each of the elements?

Reply:
This seemed to not have an effect. Exception was still thrown

Reply:
Wow you are right. https://stackoverflow.com/questions/40487944/parsing-xml-kotlin-android
@pfent this is really dumb.

@anselm92 could you write a unit test for this as well then? Just to make sure this won't break again in the future.